src/main/java/uk/gov/companieshouse/company_appointments/CompanyAppointmentService.java

```diff
@@ -55,13 +55,21 @@ public AllCompanyAppointmentsView fetchAppointmentsForCompany(String companyNumb
         }
 
         List<CompanyAppointmentView> companyAppointmentViews = allAppointmentData.stream().map(companyAppointmentMapper::map).collect(Collectors.toList());
-        int activeCount = (int) companyAppointmentViews.stream().filter(officer -> officer.getResignedOn() == null).count();
+
+        int activeCount = 0;
+        int inactiveCount = 0;
+
+        if (allAppointmentData.get(0).getCompany_status().equals("active")){
```

Comment:
We don't know for sure that "active" is the only active company status. For example there are in liquidation statuses. It would be better to check against the three known inactive statuses.